### PR TITLE
Fix ares-device/log help and error message

### PR DESCRIPTION
**Pull request by YoungeunKim**
:Release Notes:
Fix ares-device and log help and error message

:Detailed Notes:
- Update device and log help 
- Update error message and unit test statements
- Handle luna call failure in resource-monitor --list operation

:Testing Performed:
1. Passed unit test on OSE target
2. Passed ESLint
3. Verified with CLI commands
    $ ares-device -h
    $ ares-log -h (journals and pmlogd daemon)

:Issues Addressed:
[WRN-11263] Review resource-monitor/log help statements

### CODE REVIEW

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1:

File: files/help/ares-log-pmlogd.help [hunk 1]
```diff
@@ -13,25 +13,25 @@
         "This command shows or saves logs of webOS apps and services using pmlogd.",
         "",
         "** Attributes **",
-        "  LINE           [integer]  Number of lines",
-        "  LOG_DAEMON     [string]   Name of log daemon (e.g., pmlogd, journald)",
+        "  LINE           [integer]  Number of lines to show",
+        "  LOG_DAEMON     [string]   Name of logging daemon (e.g., pmlogd, journald)",
```
Comment (ssuminahn) [inline files/help/ares-log-pmlogd.help:17]: LOGGING_DAEMON

File: files/help/ares-log.help [hunk 2]
```diff
@@ -27,25 +27,25 @@
...
         "** Attributes **",
-        "  LINE         [integer]  Number of lines",
+        "  LINE         [integer]  Number of lines to show",
         "  PRIORITY     [string]   Priority of logs (e.g., info, notice, warning, err)",
         "  PID          [integer]  Process ID",
         "  DATE         [string]   Date format (e.g., YYYY-MM-DD hh:mm:ss)",
         "  UNIT         [string]   One of the unit names from --unit-list",
         "  JOURNAL_FILE [string]   One of the file names from --file-list",
-        "  LOG_DAEMON   [string]   Name of log daemon (e.g., journald, pmlogd)",
+        "  LOG_DAEMON   [string]   Name of logging daemon (e.g., journald, pmlogd)",
```
Comment (ssuminahn) [inline files/help/ares-log.help:36]: LOGGING_DAEMON

**Review by ssuminahn — COMMENTED** [review]
code :-1: 
verify :+1: 

I think it would be better to change LOG_DAEMON to LOGGING_DAEMON.

File: files/help/ares-log-pmlogd.help (hunk 1 above)
Comment (YoungeunKim) [reply files/help/ares-log-pmlogd.help]: Fixed.

File: files/help/ares-log.help (hunk 2 above)
Comment (YoungeunKim) [reply files/help/ares-log.help]: Fixed

**Comment by YoungeunKim** [comment]
code 👍 
verify 👍

**Comment by YoungeunKim** [comment]
@ssuminahn , I removed 2 spaces in Attributes items.
Code 👍 
Verifiy 👍

**Review by ssuminahn — APPROVED** [review]
code :+1: 
verify :+1:

**Pull request merged**